src/Admin/MenuBuilderInterface.php

```diff
@@ -29,7 +33,7 @@ interface MenuBuilderInterface
      *
      * @return ItemInterface|bool
      *
-     * @deprecated Use buildTabMenu instead
+     * @deprecated since sonata-project/admin-bundle 3.x
```

Comment:
This was deprecated long time ago, it would be nice to set the proper version not `3.x`.